dataStandard_versioned_docs/version-6/whats-new/whats-new-v60.md

```diff
@@ -1,441 +1,249 @@
+
+* The StudentEducationOrganizationAssociation (SEOA) was significantly reduced by the number of attributes, with demographic and directory attributes migrated to new entities.
+
+* The following new entities were created to host the demographic and contact information:
+  * StaffDemographic
+  * StaffDirectory
+  * StudentDemographic
+  * StudentDirectory
+
+* New optional entities were created to handle multiple identification codes:
+  * CandidateIdentificationCode
+  * ContactIdentificationCode
+  * EducationOrganizationIdentificationCode
+  * StaffIdentificationCode
+  * StudentIdentificationCode
+* Consequently, the StudentIdentificationCode (_optional_ common) was removed from the SEOA and became its own entity.
+
+* The Assessment, ObjectiveAssessment and StudentAssessment entities were updated (no key structural changes were introduced).
+
+* Five new domains were created to host the EPDM (Community version) entities and descriptors.  Existing Staff and Enrollment domains were enhanced to incorporate EPDM-related entities. In addition, EPDM properties previously marked as deprecated were removed.  See details below.
 
-Data Standard v5.0 is targeted at deployment starting in the 2024-25 school
-year. This final release version (v5.0.0) introduces important updates that
-impact multiple domains of the Ed-Fi data model. The domains impacted include:
-
-* Alternative and Supplemental Services Domain with Federal Programs Subdomain
-* Discipline Domain
-* Enrollment Domain
-* Graduation Domain
-* Intervention Domain
-* Student Academic Record Domain
-* Student Attendance Domain
-* Student Cohort Domain
-* Student Identification and Demographics Domain
-* Survey Domain
-* Teaching and Learning Domain
-
-These changes come from field feedback and are designed to help drive the
-community and community innovation forward. There are breaking changes in these
-domains, so implementers are advised to review the changes carefully.
-
-Other areas of the model have received minor, non-breaking updates, and API
-bindings and implementations in those areas should continue to function as
-before.
+* Two IEP date fields from the StudentSpecialEducationProgramAssociation were renamed, and two optional attributes were added to improve flexibility.
+
+:::info
+
+Additional details related to the changes for identification and demographics can be found in [Ed-Fi RFC 27b](https://edfi.atlassian.net/wiki/spaces/rc/pages/998375429/Ed-Fi+RFC+27+b+-+Streamlining+Access+to+Identification+Codes+Contact+Information+and+Demographics).
+
+Attributes decoupled from SEOA were _removed from the SEOA_, rather than deprecated, and _migrated to the new demographic and directory entities_. This approach was reviewed in multiple governance meetings hosted by the Ed-Fi Alliance and endorsed to promote consistency in data standard versioning and to reduce data redundancy.
```

Comment:
We need to avoid using acronyms when possible as a best practice.

```suggestion
Attributes decoupled from StudentEducationOrganizationAssociation, Staff, and other entities were _removed _, rather than deprecated, and _migrated_ to the new demographic and directory entities_ This approach was reviewed in multiple governance meetings hosted by the Ed-Fi Alliance and endorsed to promote consistency in data standard versioning and to reduce data redundancy.
```